.circleci/config.yml

```diff
@@ -796,6 +839,17 @@ jobs:
           # some native-dependent tests fail because of the lack of native
           # headers on emsdk-bundled clang
           test_targets: "other skip:other.test_native_link_error_message"
+  test-browser-webview:
+    executor: bionic
+    environment:
+      EMTEST_LACKS_WEBGPU: "1"
+    steps:
+      - run-tests-webview:
+          title: "browser"
+          # Skip test_4gb_fail as it OOMs on the current bot
+          test_targets: "
+            browser skip:browser.test_4gb_fail
```

Comment:
Do we want to be running all browser tests in this mode or should we just run a select few?

Reply:
As this entire suite is large (and as a result slow) I would support picking a select few to run.